src/app/components/program-details/program-details.css

```
@@ -353,8 +359,12 @@ img {
 }
 
 .partner-logo {
-    max-width: 80px;
-    height: 70px;
+    width: 120px;
+    height: 80px;
+    object-fit: contain;
+    display: block;
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: crisp-edges; 
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

image-rendering is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)